### PR TITLE
Set title properly on Move/Copy/File on Case popup

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
When clicking "Move to Case"/"Copy to Case" from the Manage cases activity list the title is initially set correctly but then always reverts to "File on Case".

Before
----------------------------------------
Incorrect title on popup.

After
----------------------------------------
Correct title on popup.

Technical Details
----------------------------------------
Initially set by javascript but then loaded via quickform defaults.

Comments
----------------------------------------
@lcdservices @alifrumin A "simple" cases UI one if you have time?

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
What does focaction mean?

**Comment by mattwire** [comment]
@eileenmcnaughton focaction = FileOnCaseAction.  It's called "action" in the javascript functions but that conflicts with the quickform "action".  I'm open to naming ideas..

**Comment by eileenmcnaughton** [comment]
@mattwire honestly I prefer words  - e.g file_on_case_action - or just file_action

File: CRM/Case/Form/ActivityToCase.php [hunk 1]
```diff
@@ -48,6 +48,15 @@ public function preProcess() {
     $this->_currentCaseId = CRM_Utils_Request::retrieve('caseId', 'Positive');
     $this->assign('currentCaseId', $this->_currentCaseId);
     $this->assign('buildCaseActivityForm', TRUE);
+
+    switch (CRM_Utils_Request::retrieve('focaction', 'String')) {
+      case 'move':
+        CRM_Utils_System::setTitle(ts('Move to Case'));
+        break;
```
Comment (yashodha) [inline CRM/Case/Form/ActivityToCase.php]: @mattwire Case breaking statements must be followed by a single blank line

**Comment by yashodha** [comment]
@mattwire Can you please check the style warning.
I agree with @eileenmcnaughton  may be case_action or file_action

**Comment by lcdservices** [comment]
I reproduced the issue and confirmed the patch fixes it. Once the style issues and variable naming is fixed, should be fine to merge.

File: CRM/Case/Form/ActivityToCase.php (hunk 1 above)
Comment (mattwire) [reply CRM/Case/Form/ActivityToCase.php]: Thankyou, fixed.

**Comment by mattwire** [comment]
@eileenmcnaughton I've made the required changes here - ok to merge?

**Comment by eileenmcnaughton** [comment]
I'm not sold on the camel case in a url var - but it's clear & greppable so I'm merging

**Pull request merged**